lib/plugins/login/login.js

```diff
@@ -1,29 +1,27 @@
 'use strict';
 
 const BbPromise = require('bluebird');
-const crypto = require('crypto');
-const readline = require('readline');
-const fetch = require('node-fetch');
 const jwtDecode = require('jwt-decode');
 const chalk = require('chalk');
+const uuid = require('uuid');
+const has = require('lodash/has');
+const forge = require('node-forge');
+const querystring = require('querystring');
 const openBrowser = require('../../utils/openBrowser');
 const configUtils = require('../../utils/config');
 const clearConsole = require('../../utils/clearConsole');
 const userStats = require('../../utils/userStats');
 const setConfig = require('../../utils/config').set;
+const createApolloClient = require('../../utils/createApolloClient');
+const decryptToken = require('./lib/decryptToken');
+const getCliLoginById = require('./lib/getCliLoginById');
 
 const config = {
-  AUTH0_CLIENT_ID: 'iiEYK0KB30gj94mjB8HP9lhhTgae0Rg3',
-  AUTH0_URL: 'https://serverlessinc.auth0.com',
-  AUTH0_CALLBACK_URL: 'https://serverless.com/auth',
+  PLATFORM_FRONTEND_BASE_URL: 'https://platform.serverless.com/',
```

Comment:
Not sure about the last `/` here.

With that I get the following URL (notice the two `//` after the `.com` part): https://platform.serverless.com//login?cli=v1&login-id=WAWS

It's not a bug per-se but could turn into problems later on.

Reply:
that's resolved with a change at another point